src/udev/net/link-config.c

```diff
@@ -937,6 +937,37 @@ static int link_apply_sr_iov_config(Link *link, sd_netlink **rtnl) {
         return 0;
 }
 
+static int link_apply_rps_cpu_mask(Link *link) {
+        _cleanup_free_ char *mask_str = NULL;
+        _cleanup_closedir_ DIR *d = NULL;
+        LinkConfig *config;
+        int r;
+
+        assert(link);
+        config = ASSERT_PTR(link->config);
+
+        mask_str = cpu_set_to_mask_string(&config->rps_cpu_mask);
+        if (!mask_str)
+                return -ENOMEM;
```

Comment:
this should use log_oom().

we genreally want that all errors are logged exactly once. for that we want that functions either log about all errors or about none (in which case the caller can do this). in this case here so far nothing logs the oom situation, but we really should